src/models/registrationExample.js

```diff
@@ -0,0 +1,9 @@
+const User = {
+  username: 'Attila',
+  password: 'xxxx5555',
+  confirmPsw: 'xxxx5565',
+};
```

Comment:
This is not a model, this is a dummy entity. It doesn't mean it's wrong, it means it's in the wrong place.

Reply:
Created a new directory for dummy. It has DB too.